package.json

```diff
@@ -1,16 +1,50 @@
 {
-  "name": "dddforum-main",
+  "name": "@dddforum/main",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "license": "ISC",
+  "workspaces": [
+    "packages/shared",
+    "packages/backend",
+    "packages/frontend"
+  ],
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=7.0.0"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "clean": "npm run clean --workspaces --if-present",
+    "build": "npm run build --workspaces --if-present",
+    "build:ci:backend": "npm run build --workspace @dddforum/shared && npm run build --workspace @dddforum/backend",
+    "build:ci:frontend": "npm run build --workspace @dddforum/shared && npm run build --workspace @dddforum/frontend",
+    "start:ci:backend": "npm run start:ci --workspace @dddforum/backend",
```

Comment:
It fails as I mentioned in https://discord.com/channels/997940575590416435/1087522996094247033/1128906430284570734.

```
Error: Cannot find module '@dddforum/shared/src/config/appConfig.shared'
Require stack:
- C:\Users\kirill\Documents\wp\ddd-forum-bp-first\packages\backend\dist\index.js
```

Because of changed TS mappings in https://github.com/stemmlerjs/ddd-forum-bp-first/pull/5/commits/70c80cd052d489657345b5313a1575bb29036a63.

Reply:
Found another way, added here https://github.com/stemmlerjs/ddd-forum-bp-first/pull/5/commits/33e64a930c3560c363e72eed0b319b2caf2cdc96. 

This approach is more explicit but a little bit hacky.

Reply:
To test:

- Make sure your env satisfies `packages\backend\.env.staging` (you simply can tweak the file to point to the local PosgreSQL)
- `npm ci`
- `npm run build:ci:backend`
- `npm start:ci:backend`